metricbeat/module/ceph/osd_df/_meta/fields.yml

```diff
@@ -0,0 +1,40 @@
+- name: osd_df
```

Comment:
Left the comments in the fields.asciidoc, but they should obviously go here.

Reply:
I get it. :)